modules/apps/calendar/calendar-web-test/src/testFunctional/tests/CalendarScheduler.testcase

```diff
@@ -158,31 +157,29 @@ definition {
 
 		CalendarEvent.viewSiteCalendarEventWithDefaultPermissions(
 			calendarName = ${siteName},
-			endDateTime = "04:00 AM",
+			endDateTime = "4:00 AM",
 			eventTitle = "Liferay Calendar Event Title",
 			siteName = ${siteName},
-			startDateTime = "03:00 AM");
+			startDateTime = "3:00 AM");
 
 		CalendarEvent.viewSiteCalendarEventNotPresent(
-			endDateTime = "09:00 PM",
+			endDateTime = " 9:00 PM",
 			siteName = "Test Test",
-			startDateTime = "08:00 PM");
+			startDateTime = " 8:00 PM");
 
-		Navigator.gotoSpecificURLPage(
+		Navigator.gotoSitePage(
 			pageName = "My Profile",
-			url = "localhost:8080/web/test");
+			siteName = "Test");
 
-		CalendarEvent.viewSiteCalendarEventWithDefaultPermissions(
-			calendarName = "Test Test",
-			endDateTime = "09:00 PM",
-			eventTitle = "Test Test Calendar Event Title",
-			siteName = "Test Test",
-			startDateTime = "08:00 PM");
+		CalendarEvent.viewSiteCalendarEventNotPresent(
+			endDateTime = " 9:00 PM",
+			siteName = ${siteName},
+			startDateTime = " 8:00 PM");
```

Comment:
any reason why this was changed from asserting the event is there to asserting it not being present? @JerryAdrianoo @Luizcarlosqueiroz

Reply:
It's because the test was wrong, a guest user cannot see private events. I talked to Ana Buchmann and we concluded that.

Reply:
@JerryAdrianoo gotcha, do you mind writing a description in the ticket? It could help us also when we refactor this test in the future